test_user_llamaindex_example.py

```
@@ -0,0 +1,555 @@
+        print(f"   Result: {result['result']}")
+        print(f"   Tracked variables: {list(result['tracked_values'].keys())}")
+        return True
+        
+    except Exception as e:
+        print(f"❌ User's MathAgent (Async) test failed: {str(e)}")
+        return False
+    finally:
+        try:
+            os.unlink(test_file)
+        except:
+            pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)